lib/options.js

```diff
@@ -4,6 +4,8 @@ module.exports = {
   appleCallbackPath: '/users/auth/apple/callback',
   facebookPath: '/users/auth/facebook',
   facebookCallbackPath: '/users/auth/facebook/callback',
+  googlePath: '/users/google/facebook',
```

Comment:
know this isn't quite ready for review! but this might be a bug during testing ("/facebook") so wanted to mention